src/operator/tensor/elemwise_sum.h

```diff
@@ -37,6 +40,18 @@
 namespace mxnet {
 namespace op {
 
+struct ElementWiseSumParam : public dmlc::Parameter<ElementWiseSumParam> {
+  int num_args;
+  DMLC_DECLARE_PARAMETER(ElementWiseSumParam) {
+    DMLC_DECLARE_FIELD(num_args).set_lower_bound(1).set_default(2)
+        .describe("Number of inputs to be summed.");
+  }
+
+  bool operator==(const ElementWiseSumParam& other) const {
+    return this->num_args == other.num_args;
+  }
+};
+
```

Comment:
why do you create a new parameter? You don't have to use MKLDNNParamOpSign. You can use MKLDNNOpSignature.

Reply:
The parameter definition is moved from src/operator/tensor/elemwise_sum.cc to this header file in order to be used by  mkldnn_sum.cc, in order to support "add_n"(alias "ElementWiseSum") OP which has valid param, I used MKLDNNParamOpSign.